<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Import DXF file</string>
  </property>
  <widget class="QLabel" name="lblFilename">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C:\Users\arxit\Documents\test.shp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImport">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>580</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveConfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Layers mapping</string>
   </property>
   <widget class="QTableWidget" name="tabLayersMapping">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Fields mapping</string>
   </property>
   <widget class="QLabel" name="lblCurrentFieldMapping">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Click on a layer mapping to edit its field mapping</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabFieldsMapping">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnImport</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_launchImport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>605</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveConfig</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoadConfig</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>_loadConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabLayersMapping</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>Dialog</receiver>
   <slot>_tabLayersMappingCellChanged(int,int,int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>610</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_selectedLayerIndexChanged(int)</slot>
  <slot>_launchImport()</slot>
  <slot>_saveConfig()</slot>
  <slot>_loadConfig()</slot>
  <slot>_tabLayersMappingCellChanged(int,int,int,int)</slot>
 </slots>
</ui>
